doc/blocks.xml

```diff
@@ -173,3 +173,27 @@ gap> DegreeOfBlocks(blocks);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="CanonicalBlocks">
+  <ManSection>
+    <Attr Name = "CanonicalBlocks" Arg = "blocks"/>
+    <Returns>Blocks of a bipartition.</Returns>
+    <Description>
+      For the blocks of a bipartition <A>blocks</A> the function
```

Comment:
When I read this, it made it seem to me that the argument is a bipartition. Maybe reword this
"If <A>blocks</A> is the blocks of a bipartition, then ..."